### PR TITLE
code challenge 17 complete

### PR DESCRIPTION
**Pull request by mparkersimms**
[ x] Top-level README “Table of Contents” is updated
[x ] Feature tasks for this challenge are completed
[ x] Unit tests written and passing
[x ] “Happy Path” - Expected outcome
[ x] Expected failure
[ x] Edge Case (if applicable/obvious)
[ x] README for this challenge is complete
[ x] Summary, Description, Approach & Efficiency, Solution
[ x] Link to code
[x ] Picture of whiteboard

### CODE REVIEW

File: utilities/app/src/test/java/utilities/Tree/TreeTester.java [hunk 1]
```diff
@@ -98,4 +98,17 @@
...
+    @Test public void breadthFirstTest() throws Exception {
+        Tree testTree2 = new Tree();
+        testTree2.root = new Node(1, null, null);
+        testTree2.root.left = new Node(2, null, null);
+        testTree2.root.right = new Node(3, null, null);
+        testTree2.root.left.left = new Node(8, null, null);
+        testTree2.root.left.right = new Node(5, null, null);
+        testTree2.root.right.left = new Node(9, null, null);
+
+        System.out.println(testTree2.breadthFirstTraversal());
```
Comment (Mattpet26) [inline utilities/app/src/test/java/utilities/Tree/TreeTester.java:110]: A sout isn't the same as an assertion. You can compare arrays by doing AssertArraysEquals
